### ISSUE TITLE
Does not seem to be working with Cura 5.0

### ISSUE THREAD

**Issue by CaryBab** [fieldOfView/Cura-CustomJobPrefix#33]
I don't know if it is a problem with the new version of Cura, which is what I suspect.

**Comment by fieldOfView** [comment]
Have you updated the plugin to the latest version (3.7.0)?

If so, and it is still not working, try the following:
* Uninstall the plugin via the Marketplace "package manager" tab (the "cogwheel" icon after Plugins and Materials)
* Restart Cura
* Reinstall the plugin via the Marketplace
* Restart Cura again

**Comment by CaryBab** [comment]
Yes it was 3.7.0.  It does display the name like it should in the lower
left, but Cura does not use it when I save the file.  It doesn't fill out
any name, I have to completely fill it out.  I think this is a Cura issue
running on Linux.  I will stick to 4.13.1 until things are working better.

**Comment by fieldOfView** [comment]
>  It doesn't fill out any name, I have to completely fill it out. 

I don't understand what you mean. Can you tell me exactly what you have filled in in the job name dialog, and what happens when saving a file?

**Comment by CaryBab** [comment]
When I load a file into Cura 5.0, The filename in the bottom left displayed
by CustomPrintJob has all the correct stuff but everything is greyed out
except the actual name of what I loaded.

When I save the sliced object, Cura brings up the screen to save the file,
but has a blank filename to save, not even the basic name of the file I
loaded, and I have to completely fill out the filename to save it.

I hope that explains it better.  I believe it is a cura problem since the
CustomJobName is displaying, although it is greyed out.

Cary

**Comment by CaryBab** [comment]
Be aware I am running Linux, so I assume it works in windows, since it
seems to be working for most people.

**Comment by PoloClive10** [comment]
I am having the same issue. Using Cura on Mac though. I have to manually edit the name every time. I also can't seem to find where to check the settings for the plugin either.
